packages/webdriverio/src/commands/element/nextElement.js

```diff
@@ -0,0 +1,27 @@
+
+/**
+ *
+ * Returns the next sibling element of the selected DOM-element.
+ *
+ * <example>
+    :index.html
+    <div class="parent">
+        <p>Sibling One</p>
+        <p>Sibling Two</p>
+        <p>Sibling Three</p>
+    </div>
+    :nextElement.js
+    it('should get text from next sibling element', () => {
+        const elem = $$('p');
+        console.log(elem[1].nextElement().getText()); // outputs: "Sibling Three"
+    });
+ * </example>
+ *
+ * @alias element.nextElement
+ * @return {Element}
+ * @type utility
+ */
+
+export default function nextElement () {
+    return this.$(/* istanbul ignore next */ function () { return this.nextElementSibling })
```

Comment:
@christian-bromann I had to revert to function () here due to () => not working in Mocha - is this due to ES6 compatibility ?

Reply:
not sure but it is fine like this